packages/angular/cli/commands/update-impl.ts

```diff
@@ -189,9 +189,9 @@ export class UpdateCommand extends Command<UpdateCommandSchema> {
 
       // Commit migration
       if (commit) {
-        let message = `migrate workspace for ${packageName}@${migration.version}`;
+        let message = `${packageName}@${migration.version} workspace migration - ${migration.name}.`;
```

Comment:
I think we should remove the version.  They are typically set to values that are not very useful to the end user.
Maybe drop “workspace” as well for brevity.

Reply:
Yeah this is a perfect use case for the the `title` field.